crates/sui-core/src/authority/shared_object_congestion_tracker.rs

```diff
@@ -992,6 +1095,7 @@ mod object_cost_tests {
                     &tx,
                     &HashMap::new(),
                     0,
+                    Duration::from_micros(10_000_000),
```

Comment:
worth adding a test where you set this to something other than 10s and make sure it works as expected?